tests/TestOne/buildtool/bin/buildtool

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -e
+#
+# This is a very simple pjc-aware ocamlfind-based build tool.
+
+# Build
+
+DEPENDS=""
+for p in $cur_depends; do
+  if [ "$p" != "buildtool" ]; then
+    DEPENDS="$DEPENDS $p"
+  fi
+done
+
+echo "Build location: $cur_target_dir"
+
+mkdir -p $cur_target_dir/lib
+mkdir -p $cur_target_dir/bin
+
+## Prepare ocamlfind options
+
+OCAMLFIND_ARGS=""
+
+for p in $DEPENDS; do
+  OCAMLFIND_ARGS="$OCAMLFIND_ARGS -package $p"
+done
+
+## Build modules
+
+for src in `find $cur_root/lib -name "*.ml"`; do
+  ocamlfind ocamlc -c $OCAMLFIND_ARGS $src -o $cur_target_dir/lib/`basename $src`
+done
+
+for src in `find $cur_root/bin -name "*.ml"`; do
+  ocamlfind ocamlc -c $OCAMLFIND_ARGS $src -o $cur_target_dir/bin/`basename $src`
+done
+
+## Link executables
+
+for obj in `find $cur_target_dir/bin -name "*.cmo"`; do
+  target=$cur_target_dir/`basename ${src%.*}`
+  ocamlfind ocamlc \
+    -linkpkg $OCAMLFIND_ARGS \
+    $obj -o $target
+done
+
+## Produce archive
+
+ocamlc -a -o $cur_target_dir/lib/$cur_name.cma `find $cur_target_dir/lib -name "*.cmo"`
+
+## Write META for findlib installation
+
+cat <<EOF > "$cur_target_dir"/lib/META
+pkgname = "$cur_name"
+version = "$cur_version"
+archive(byte) = "$cur_name.cma"
+requires = "$DEPENDS"
+EOF
+
+# Install
+
+echo "Install location: $cur_install"
+
+mkdir -p $cur_install
```

Comment:
Same here (`cur__install`).
This is totally the right way to do this in general though. This is a general purpose build script that needs to work dynamically on the "cur" package - whichever that may be!